packages/next/build/swc/index.js

```diff
@@ -74,7 +74,8 @@ async function loadWasm() {
           return Promise.resolve(bindings.minifySync(src.toString(), options))
         },
         parse(src, options) {
-          return Promise.resolve(bindings.parse(src.toString(), options))
+          const astStr = bindings.parseSync(src.toString(), options)
```

Comment:
Does this need to use `bindings.parse` when not using the wasm version or should both bindings have `parse` exposed? x-ref: https://github.com/vercel/next.js/blob/743cf685555e680ae539844f317c03c806028d7a/packages/next-swc/crates/napi/src/lib.rs#L73

Reply:
I think the native binding is [here](https://github.com/vercel/next.js/pull/36482/files#diff-b3bf60b472b7bb06493a547602dc446d86a0e0f08f8a3b0c96662ae23fd95629R191-R193)? and previously this doesn't work since we don't have a `parse` or `parseSync` api in wasm

Reply:
The link isn't highlighting the exact line for some reason, but @huozhi is correct it is setup farther down in this file 👍